jina/peapods/__init__.py

```diff
@@ -76,24 +72,51 @@ def Pod(args: Union['argparse.Namespace', Dict] = None, allow_remote: bool = Tru
                 from .pods.mutable import MutablePod
                 return MutablePod(args)
             else:
-                # TODO: this part needs to be refactored
-                from .pods.remote import RemoteMutablePod
-                return RemoteMutablePod(args)
+                from .runtimes.remote.jinad import RemoteJinaDRunTime
+                return RemoteJinaDRunTime(args, kind='pod')
 
     if not allow_remote and args.host != __default_host__:
         args.host = __default_host__
         default_logger.warning(f'host is reset to {__default_host__} as allow_remote=False')
 
     if args.host != __default_host__:
         if args.remote_access == RemoteAccessType.JINAD:
-            from .pods.remote import RemotePod
-            return RemotePod(args)
+            from .runtimes.remote.jinad import RemoteJinaDRunTime
+            return RemoteJinaDRunTime(args, kind='pod')
         elif args.remote_access == RemoteAccessType.SSH:
-            from .peas.remote.ssh import RemoteSSHPod
-            return RemoteSSHPod(args)
+            from .runtimes.remote.ssh import RemoteSSHRunTime
+            return RemoteSSHRunTime(args, kind='pod')
         else:
             raise ValueError(f'{args.remote_access} is not supported')
-
     else:
         from .pods import BasePod
         return BasePod(args)
+
+
+def Pea(args: Optional['argparse.Namespace'] = None,
```

Comment:
Should we move the `Pea` method outside `peapods.__init__`, since this is not invoked by the api anymore?